src/myspace/MyPersonalDetails.jsx

```diff
@@ -0,0 +1,288 @@
+import React, { useState } from "react";
+import { BASE_API } from "../config";
+import Cookies from "js-cookie";
+import "./MySpace.css";
+
+
+export default function MyPersonalDetails() {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [username, setUsername] = useState(null);
+  const [email, setEmail] = useState(null);
+  const [name, setName] = useState(null);
+  const [ircId, setIrcId] = useState(null);
+  const [mentor, setMentor] = useState(null);
+  const [mentee, setMentee] = useState(null);
+  const [interests, setInterests] = useState(null);
+  const [bio, setBio] = useState(null);
+  const [location, setLocation] = useState(null);
+  const [occupation, setOccupation] = useState(null);
+  const [currentOrganization, setCurrentOrganization] = useState(null);
+  const [skills, setSkills] = useState(null);
+  const [resumeUrl, setResumeUrl] = useState(null);
+  const [photoUrl, setPhotoUrl] = useState(null);
+
```

Comment:
why not just have const [personalDetails, setPersonalDetails] = useState(null);

and set whatever comes in response as the personalDetails? 

I don't see the benefit in cluttering with so many use states

Reply:
@meenakshi-dhanani , I tried your suggestion. Using const [personalDetails, setPersonalDetails] = useState(null) or useState({}) but none of these options get the data out from the response. Here's the screenshot of what I've done.
<img width="562" alt="Screen Shot 2020-07-13 at 12 27 26 am" src="https://user-images.githubusercontent.com/29667122/87249430-61a26480-c4a2-11ea-91ce-132ae52e9243.png">

<img width="1680" alt="Screen Shot 2020-07-13 at 12 42 50 am" src="https://user-images.githubusercontent.com/29667122/87249459-8eef1280-c4a2-11ea-8005-6b41ed31ca9d.png">

Do you have any other suggestion?

Reply:
And here's the result of thee page when rendered (with empty data)
<img width="1680" alt="Screen Shot 2020-07-13 at 12 43 46 am" src="https://user-images.githubusercontent.com/29667122/87249634-83501b80-c4a3-11ea-9904-912590017368.png">

Reply:
I'll check it out, as a convention if your setter is setPersonalDetails then your variable would be personalDetails and not personal_details

Reply:
Hi @meenakshi-dhanani . I tried with the correct naming convention. Still didn't work (cannot get data out of fetch using setPersonalDetails)
* using null
<img width="651" alt="Screen Shot 2020-07-14 at 11 24 07 am" src="https://user-images.githubusercontent.com/29667122/87370143-86432d00-c5c5-11ea-837a-ab34919fe430.png">
<img width="1560" alt="Screen Shot 2020-07-14 at 11 25 35 am" src="https://user-images.githubusercontent.com/29667122/87370154-8e02d180-c5c5-11ea-93c4-bb5aaccb3248.png">

* using {}
<img width="634" alt="Screen Shot 2020-07-14 at 11 27 53 am" src="https://user-images.githubusercontent.com/29667122/87370179-9955fd00-c5c5-11ea-8179-1e5b5f066ae0.png">
<img width="1578" alt="Screen Shot 2020-07-14 at 11 28 03 am" src="https://user-images.githubusercontent.com/29667122/87370230-b38fdb00-c5c5-11ea-9545-335a5f75b21a.png">

Do you have any other suggestion?

Reply:
Check the Todo file in this sandbox - https://codesandbox.io/s/xenodochial-rain-fet6s?file=/src/Todo.js

Here, I am not setting time and title separately.

Reply:
@meenakshi-dhanani . Have you try it using the fetch from API? Does it work on your end? I've pushed the latest code with your alternative solution in case you want to give it a try. Let me know if it works for you. If you want to catch up for troubleshoot, I'm available. Ta